testapi.pm

```diff
@@ -507,7 +507,13 @@ sub click_lastmatch (%args) {
     # Calculate the absolute click point.
     my ($x, $y) = _calculate_clickpoint($last_matched_needle, $relevant_area, $relative_click_point);
     bmwqemu::diag("clicking at $x/$y");
-    mouse_set($x, $y);
+    if ($bmwqemu::vars{DEMO_MODE}) {
```

Comment:
the variable "DEMO_MODE" does a similar thing to the variable "NICEVIDEO" in osado, see https://github.com/search?q=repo%3Aos-autoinst%2Fos-autoinst-distri-opensuse%20NICEVIDEO&type=code